### PR TITLE
Add TongYiChatOptions repetitionPenalty parameter build

### PR DESCRIPTION
**Pull request by zhangqian9158**
### Describe what this PR does / why we need it

Add TongYiChatOptions repetitionPenalty parameter build

### CODE REVIEW

File: spring-cloud-alibaba-examples/rocketmq-example/rocketmq-broadcast-example/rocketmq-broadcast-consumer1-example/src/main/resources/application.yml [hunk 1]
```diff
@@ -4,9 +4,9 @@ spring:
   application:
     name: rocketmq-broadcast-consumer1-example
   cloud:
+    function:
+      definition: consumer;
     stream:
-      function:
```
Comment (yuluo-yx) [inline spring-cloud-alibaba-examples/rocketmq-example/rocketmq-broadcast-example/rocketmq-broadcast-consumer1-example/src/main/resources/application.yml]: why changed this?

Comment (zhangqian9158) [reply spring-cloud-alibaba-examples/rocketmq-example/rocketmq-broadcast-example/rocketmq-broadcast-consumer1-example/src/main/resources/application.yml]: rocketMQ related changes have been restored and submitted in another PR

**Comment by yuluo-yx** [comment]
image ？

**Comment by zhangqian9158** [comment]
@yuluo-yx

**Comment by yuluo-yx** [comment]
可以在参数转换的时候也加入这个参数进去，现在只是提供了配置项，并没有用到SDK请求参数离去，可以参考这里： 
https://github.com/alibaba/spring-cloud-alibaba/blob/79bca1eb42a328abe954f47676b1b60437b5ad2f/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-ai/src/main/java/com/alibaba/cloud/ai/tongyi/chat/TongYiChatClient.java#L224

**Comment by zhangqian9158** [comment]
企业微信截图_1718247466308
ConversationParam parameter conversion already exists

**Review by yuluo-yx — APPROVED** [review]
LGTM. tks @zhangqian9158

**Pull request merged**